packages/renderer/src/hookConsole.ts

```diff
@@ -0,0 +1,50 @@
+
+import { Hierarchy, Logger, HierarchyLog, StringLog, LogLevel } from "@escad/core"
+
+export const hookConsole = (console: Console, logger: Logger) => {
+  const log = console.log.bind(console)
+  console.log = (...args: unknown[]) => {
+    log(...args)
+    standardLog(args, "log")
+  }
+
+  const warn = console.warn.bind(console)
+  console.warn = (...args: unknown[]) => {
+    warn(...args)
+    standardLog(args, "warn")
+  }
+
+  const error = console.error.bind(console)
+  console.error = (...args: unknown[]) => {
+    error(...args)
+    standardLog(args, "error")
+  }
+
+  const clear = console.clear.bind(console)
+  console.clear = () => {
+    clear()
+    logger.log(null)
+  }
+
+  const timeEnd = console.timeEnd.bind(console)
+  console.timeEnd = (label?: string) => {
+    const overridenLog = console.log
+    console.log = log
+    timeEnd(label)
+    console.log = overridenLog
+  }
```

Comment:
```suggestion
  const consoleLog = console.log.bind(console)
  const overridenLog = console.log = (...args: unknown[]) => {
    consoleLog(...args)
    standardLog(args, "log")
  }

  const consoleWarn = console.warn.bind(console)
  console.warn = (...args: unknown[]) => {
    consoleWarn(...args)
    standardLog(args, "warn")
  }

  const consoleError = console.error.bind(console)
  console.error = (...args: unknown[]) => {
    consoleError(...args)
    standardLog(args, "error")
  }

  const consoleClear = console.clear.bind(console)
  console.clear = () => {
    consoleClear()
    logger.log(null)
  }

  const consoleTimeEnd = console.timeEnd.bind(console)
  console.timeEnd = (label?: string) => {
    console.log = consoleLog
    consoleTimeEnd(label)
    console.log = overridenLog
  }
```

Reply:
(incorporates this and one above)
```suggestion
  const consoleLog = console.log.bind(console)
  const overridenLog = console.log = (...args: unknown[]) => {
    consoleLog(...args)
    standardLog(args, "log")
  }
  const consoleWarn = console.warn.bind(console)
  console.warn = (...args: unknown[]) => {
    consoleWarn(...args)
    standardLog(args, "warn")
  }
  const consoleError = console.error.bind(console)
  console.error = (...args: unknown[]) => {
    consoleError(...args)
    standardLog(args, "error")
  }
  const consoleClear = console.clear.bind(console)
  console.clear = () => {
    consoleClear()
    logger.clear()
  }
  const consoleTimeEnd = console.timeEnd.bind(console)
  console.timeEnd = (label?: string) => {
    console.log = consoleLog
    consoleTimeEnd(label)
    console.log = overridenLog
  }
```